great_expectations/core/expectation_validation_result.py

```diff
@@ -79,6 +79,15 @@ def __eq__(self, other):
             # Delegate comparison to the other instance's __eq__.
             return NotImplemented
         try:
+            if self.result and other.result:
+                common_keys = set(self.result.keys()).intersection(other.result.keys())
```

Comment:
Could this be simplified to 
```suggestion
                common_keys = self.result.keys() & self.result.keys()
```

Reply:
@cdkini -- thanks! (will go in to the next commit).